app/assets/stylesheets/application.scss

```diff
@@ -1,8 +1,15 @@
 $alizarin-crimson: #d72130;
 
-@import 'bootstrap';
-@import "font-awesome";
+@import 'font-awesome';
+@import "bootstrap/scss/bootstrap";
```

Comment:
there was generated some commment
```
/*
 * This is a manifest file that'll be compiled into application.css, which will include all the files
 * listed below.
 *
 * Any CSS and SCSS file within this directory, lib/assets/stylesheets, or any plugin's
 * vendor/assets/stylesheets directory can be referenced here using a relative path.
 *
 * You're free to add application-wide styles to this file and they'll appear at the bottom of the
 * compiled file so the styles you add here take precedence over styles defined in any other CSS/SCSS
 * files in this directory. Styles in this file should be added after the last require_* statement.
 * It is generally better to create a new file per style scope.
 *
 *= require_tree .
 *= require_self
 */
```
which was deleted, but it was not just simple comment

` *= require_tree .` - was requiring all css files in directory
` *= require my_custom_style` - could require your custom css file

Reply:
resolved